src/foundry/client-esm/applications/api/application.d.mts

```diff
@@ -417,7 +447,7 @@ declare class ApplicationV2<
    * @returns The result of HTML rendering may be implementation specific.
    *          Whatever value is returned here is passed to _replaceHTML
    */
-  protected _renderHTML(context: ApplicationV2.RenderContext, options: DeepPartial<RenderOptions>): Promise<any>; //TODO: Might be the subject of a generic?
+  protected _renderHTML(context: RenderContext, options: DeepPartial<RenderOptions>): Promise<unknown>;
```

Comment:
`_renderHTML` is in one place an `HTMLElement` and in another a `Record<string, HTMLElement>`. It's also documented as being able to be anything arbitrary. See the above documentation for `@returns`.

If this ends up not panning out to be arbitrary it may be good to revisit this later but it'd have to be a bit of a weird union here. I see this in a similar way to `toObject` and its broad return value. It may make it harder to use but I'm not sure how much we can do about that.